crane/container.go

```diff
@@ -517,9 +535,12 @@ func (c *container) Status() []string {
 		return []string{c.prefixedName(), c.Image(), "-", "-", "-", "-", "-"}
 	}
 	fields := []string{c.ActualName(), c.Image(), "-", "-", "-", "-", "-"}
-	output := inspectString(c.Id(), "{{.Id}}+++{{.Image}}+++{{if .NetworkSettings.IPAddress}}{{.NetworkSettings.IPAddress}}{{else}}-{{end}}+++{{range $k,$v := $.NetworkSettings.Ports}}{{$k}},{{else}}-{{end}}+++{{.State.Running}}")
+
+	// When using a `--tag` global flag, c.Image() may not represent an actual image tag.
+	// Instead we should get an image tag by inspecting "Config.Image".
+	output := inspectString(c.Id(), "{{.Config.Image}}+++{{.Id}}+++{{.Image}}+++{{if .NetworkSettings.IPAddress}}{{.NetworkSettings.IPAddress}}{{else}}-{{end}}+++{{range $k,$v := $.NetworkSettings.Ports}}{{$k}},{{else}}-{{end}}+++{{.State.Running}}")
```

Comment:
Suppose we have just lifted `app` container and its image name is `nginx`. Now we can get status of app.
```
NAME	IMAGE		ID				UP TO DATE	IP			PORTS			RUNNING
app		nginx		fd2d4db87a21	true		172.17.0.3	443/tcp,80/tcp,	true
```

Then we export CRANE_TAG=1.9 and see status again without above fix.

```
NAME	IMAGE		ID				UP TO DATE	IP			PORTS			RUNNING
app		nginx:1.9	fd2d4db87a21	true		172.17.0.3	443/tcp,80/tcp,	true
```

This output is incorrect. So I think `crane status` should inspect `Config.Image` to get an actual image name rather than using an image name overridden by `--tag`.